### PR TITLE
Main window

### PR DESCRIPTION
**Pull request by Tristanas**
Made a base for the main window and both created and added functionality to several data classes.

### CODE REVIEW

**Comment by TMikna** [comment]
I Offer the next time create enother pull request for data classes data classes, overall looks not so bad for me.
Lets's wait for Darius comment, If there are some drawbacks I want to know as well.

File: SafePlace/MainWindow.xaml.cs [hunk 1]
```diff
@@ -20,9 +20,35 @@ namespace SafePlace
     /// </summary>
     public partial class MainWindow : Window
     {
+        Service.ConsoleLogger Logger = new Service.ConsoleLogger(); //might be bad practice, but currently convenient
```
Comment (TMikna) [inline SafePlace/MainWindow.xaml.cs]: It's not good practise to leave bad practices. If we start doing like "It does not look good, but ir works so I'm leaving it" later will be hard.

**Comment by Tristanas** [comment]
The current main window is far from our goal. Its contents are supposed to be resources for futher development. Take the buttons as the basis of the Top-Bar. Consider these as things to be implemented.

File: SafePlace/ViewModels/BaseViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+namespace WpfApp1.ViewModels
+{
+    class BaseViewModel : INotifyPropertyChanged
+    {
+        public event PropertyChangedEventHandler PropertyChanged = (sender, e) => { };
+        public string Test { set; get; }
+
+        public BaseViewModel()
+        {
+            Task.Run(async () =>
```
Comment (Tristanas) [inline SafePlace/ViewModels/BaseViewModel.cs]: The argumentless constructor was used to test binding. The label that was bound to the field "Test" seems to be missing from the xaml.

**Pull request merged**